### PR TITLE
feat(deps): drop node8 & update dependencies

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -72,7 +73,7 @@ let p: Promise<void | {}>;
 
 switch (cmd) {
   case 'sync-repo-settings':
-    p = syncRepoSettings(cli.flags as Flags);
+    p = syncRepoSettings((cli.flags as unknown) as Flags);
```
Comment (summer-ji-eng) [inline src/cli.ts:76]: I prefer ignore linter rather than unknow

File: src/cli.ts [hunk 2]
```diff
@@ -81,7 +82,7 @@ switch (cmd) {
     p = reconcileUsers();
     break;
   case 'issues':
-    p = showIssues(cli.flags as Flags);
+    p = showIssues((cli.flags as unknown) as Flags);
```
Comment (summer-ji-eng) [inline src/cli.ts:85]: same here.

File: src/cli.ts (hunk 1 above)
Comment (xiaozhenliu-gg5) [reply src/cli.ts:76]: Unfortunately this is not linter issue, but type incompatibility, it fails `npm test`. same as below.  :(

Comment (summer-ji-eng) [reply src/cli.ts:76]: Oh, I see. Thanks for explaining.

**Pull request merged**